### ISSUE TITLE
SyntaxError: missing ) after argument list for npm build

### ISSUE THREAD

**Issue by kowalczm** [eirslett/frontend-maven-plugin#578]
When I'm executing below pom.xml:
`	<build>
		<plugins>
			<plugin>
				<groupId>org.springframework.boot</groupId>
				<artifactId>spring-boot-maven-plugin</artifactId>
			</plugin>

			<plugin>
				<groupId>com.github.eirslett</groupId>
				<artifactId>frontend-maven-plugin</artifactId>
				<configuration>
					<nodeVersion>v7.5.0</nodeVersion>
					<npmVersion>4.1.2</npmVersion>
					<workingDirectory>src/main/ts</workingDirectory>
					<installDirectory>.node</installDirectory>
				</configuration>
				<executions>
					<execution>
						<id>install node and npm</id>
						<phase>initialize</phase>
						<goals>
							<goal>install-node-and-npm</goal>
						</goals>
					</execution>
					<execution>
						<id>npm install</id>
						<phase>initialize</phase>
						<goals>
							<goal>npm</goal>
						</goals>
						<configuration>
							<arguments>install</arguments>
						</configuration>
					</execution>
					<execution>
						<id>npm run-script mavenbuild</id>
						<phase>compile</phase>
						<goals>
							<goal>npm</goal>
						</goals>
						<configuration>
							<arguments>run-script mavenbuild</arguments>
						</configuration>
					</execution>
				</executions>
			</plugin>
		</plugins>
	</build>`
I'm getting below exception:
[ERROR] D:\git\webui-angular\src\main\ts\node_modules\.bin\ng:2
[ERROR] basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
[ERROR]           ^^^^^^^
[ERROR] SyntaxError: missing ) after argument list
[ERROR]     at Object.exports.runInThisContext (vm.js:73:16)
[ERROR]     at Module._compile (module.js:543:28)
[ERROR]     at Object.Module._extensions..js (module.js:580:10)
[ERROR]     at Module.load (module.js:488:32)
[ERROR]     at tryModuleLoad (module.js:447:12)
[ERROR]     at Function.Module._load (module.js:439:3)
[ERROR]     at Module.runMain (module.js:605:10)
[ERROR]     at run (bootstrap_node.js:418:7)
[ERROR]     at startup (bootstrap_node.js:139:9)
[ERROR]     at bootstrap_node.js:533:3
[ERROR] 
[ERROR] npm ERR! Windows_NT 6.1.7601
[ERROR] npm ERR! argv "D:\\git\\webui-angular\\.node\\node\\node.exe" "D:\\git\\webui-angular\\.node\\node\\node_modules\\npm\\bin\\npm-cli.js" "run-script" "mavenbuild"
[ERROR] npm ERR! node v7.5.0
[ERROR] npm ERR! npm  v4.1.2
[ERROR] npm ERR! code ELIFECYCLE
[ERROR] npm ERR! webui-angular@0.0.0 mavenbuild: `node node_modules/.bin/ng build`
[ERROR] npm ERR! Exit status 1
I'm using Win7.
Basically I want to execute (mvn clean package) this project: [https://github.com/eitsopensource/ngboot](url)
 Do you know what can be wrong?

**Comment by mriehema** [comment]
The problem is in your "mavenbuild" script. 
Can you run "npm run-script mavenbuild" directly (without maven)?

**Comment by kowalczm** [comment]
Yes I have the same problem when I run:
"npm run-script mavenbuild" which runs 'node node_modules/.bin/ng build'

**Comment by mriehema** [comment]
Well, then you should fix your script. Not a bug of frontend-maven-plugin.
